kernel/libs/aster-bigtcp/src/errors.rs

```diff
@@ -10,7 +10,41 @@ pub enum BindError {
 }
 
 pub mod tcp {
-    pub use smoltcp::socket::tcp::{ConnectError, ListenError, RecvError, SendError};
+    pub use smoltcp::socket::tcp::{RecvError, SendError};
+
+    #[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord)]
```

Comment:
Make the `derive` list and doc comments consistent with `udp::SendError`?

Not sure why we want `PartialOrd`/`Ord` for an error type.